generators/rails-generator/Rakefile

```diff
@@ -0,0 +1,337 @@
+        end
+      end
+    end
+    # singleton_class.class_eval in included
+    use Orthoses::Tap do |store|
+      store["ActiveRecord::ModelSchema"].body.tap do |body|
+        body.delete("alias _inheritance_column= inheritance_column=")
+        body.delete("alias inheritance_column= real_inheritance_column=")
+      end
+    end
+    # alias in included block
+    use Orthoses::Tap do |store|
+      store["ActiveRecord::ConnectionAdapters::ColumnMethods"].body.tap do |body|
+        body.delete("alias blob binary")
+        body.delete("alias numeric decimal")
+      end
+    end
+    # > Use async_exec instead of exec_params on pg versions before 1.1
+    use Orthoses::Tap do |store|
+      store["PG::Connection"].body.clear
+    end
+    # Entrust to auto super class
+    use Orthoses::Tap do |store|
+      store.each do |_, content|
+        if content.header&.include?(" < Type::")
+          content.header.sub!(/ < Type::(.*)/, " < ::ActiveModel::Type::\\1")
+        end
+        # delegate to auto_header
+        if content.header&.start_with?("class Arel")
+          content.header = nil
+        end
+      end
+      store["ActionView::Helpers::Tags::CollectionRadioButtons::RadioButtonBuilder"].header = nil
+      store["ActionView::Helpers::Tags::CollectionCheckBoxes::CheckBoxBuilder"].header = nil
+      store["ActionView::SyntaxErrorInTemplate"].header = nil
+      # MigrationProxy cannot resolve name since class alias.
+      store["ActiveRecord::NullMigration"].header = nil
+    end
+    use Orthoses::DelegateClass
+    use Orthoses::Attribute
+    use Orthoses::Railties::Mixin, callback: -> (railties_mixin) {
+      Orthoses::CreateFileByName.new(
+        ->{ railties_mixin },
+        base_dir: "#{out_dir}/#{version}/railties_mixin",
+        header: "# !!! GENERATED CODE !!!\n# Please see generators/rails-generator"
+      ).call
+    }
+    use Orthoses::Mixin,
+      if: -> (base_mod, how, mod) {
+        mod != Enumerable # TODO
+      }
+    use Orthoses::RBSPrototypeRB,
+      paths: Dir.glob('_src/{railties,action{cable,mailbox,mailer,pack,text,view},active{job,model,record,storage,support}}/{app,lib}/**/*.rb'),
+      constant_filter: -> (member) { false },
+      mixin_filter: -> (member) { false },
+      attribute_filter: -> (member) { false }
+    use Orthoses::Autoload
+    run loader
+  end.call
+
+  # $ cat out/7.0/**/*.rbs | wc
+  # 69763  339342 2606899
+end
+
+def generate_test_script(gem:, version:, export:, stdlib_dependencies:, gem_dependencies:, rails_dependencies:)
+  Pathname(export).join('_scripts').tap(&:mkdir).join('test').write(<<~SHELL)
+    #!/usr/bin/env bash
+
+    # !!! GENERATED CODE !!!
+    # Please see generators/rails-generator
+
+    # set -eou => Exit command with non-zero status code, Output logs of every command executed, Treat unset variables as an error when substituting.
+    set -eou pipefail
+    # Internal Field Separator - Linux shell variable
+    IFS=$'\n\t'
```

Comment:
Could you apply the same changes as https://github.com/ruby/gem_rbs_collection/pull/224?
It's my mistake. Sorry for the inconvenience.

Reply:
Supported 👍